manifest.json

```diff
@@ -629,6 +629,18 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/ubuntu/18.04/mlnet",
+              "os": "linux",
+              "osVersion": "bionic",
+              "tags": {
+                "ubuntu-18.04-mlnet-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
+              }
+            }
+          ]
+        },
```

Comment:
Could you place this after the `src/ubuntu/18.04/debpkg` platform so that the platforms are in alphabetical order according to Dockerfile path?

Reply:
Yep. Order has been fixed.